plugins/accelerated-peft/requirements.txt

```diff
@@ -3,11 +3,7 @@
 
 # put this in here because there is a breaking FSDP api change that 
 # is fixed after peft > 0.10
-accelerate < 0.29
+accelerate <= 0.29
```

Comment:
i think this one is resolved already, can you test if we can release this and update #17